spaces/serializers.py

```diff
@@ -0,0 +1,47 @@
+from rest_framework import serializers
+
+from .models import Address, Place
+
+
+class AddressSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Address
+        fields = ('street_name', 'civic', 'floor', 'door', 'city', 'postal_code', 'region', 'nation')
+
+
+class PlaceSerializer(serializers.HyperlinkedModelSerializer):
+    address = AddressSerializer()
+
+    class Meta:
+        model = Place
+        fields = ('url', 'name', 'owner', 'description', 'address')
+
+    def update(self, instance, validated_data):
+        instance.name = validated_data.get('name', instance.name)
+        instance.owner = validated_data.get('owner', instance.owner)
+        instance.description = validated_data.get('nationality', instance.description)
+        instance.address = validated_data.get('address', instance.address)
+        instance.save()
+        return instance
+
+    def create(self, validated_data):
+        address_data = validated_data.get('address')
+        address = Address.objects.create(street_name=address_data.get('street_name'),
```

Comment:
Then, it would be nice if both save's were performed in a single DB transaction (in case one fails so do the others).

For that, we can use [django.db.transaction.atomic()](https://docs.djangoproject.com/en/1.10/topics/db/transactions/#django.db.transaction.atomic) either as a decorator on the update/create function, or as a context inside, as

```python
with transaction.atomic():
  # stuff here
```